pkgs/by-name/op/openscad-unstable/package.nix

```diff
@@ -119,11 +79,13 @@ clangStdenv.mkDerivation rec {
     ghostscript
     glib
     gmp
+    opencsg
     harfbuzz
     hidapi
     lib3mf
     libspnav
     libzip
+    # manifold  after https://github.com/NixOS/nixpkgs/pull/342211 submitted
```

Comment:
I'm thinking if we should just wait for that?

Reply:
sounds good, let me mark this PR as Draft for now.

Reply:
manifold is merged, now using the system manifold, i.e. `USE_BUILTIN_MANIFOLD=OFF`